administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -33,7 +26,35 @@ function triggerSave() {
 			$('#left-panel').toggle();
 			$('#right-panel').toggleClass('full-width');
 		});
+
+		$('.btn-success').attr('onclick','return triggerSave()');
+
+		$('#reference-association').load(function (){
+			$('#reference-association').each(function () {
+				$(this).contents().find('.chzn-single').css('background', 'transparent');
+				$(this).contents().find('.chzn-single').css('background-color', '#eee');
+				$(this).contents().find('.controls').css( 'pointer-events', 'none' );
+    			$(this).contents().find('input').attr('disabled', 'disabled');
```

Comment:
i would recommend that as i certanly can see the use for that, but let's see @infograf768 opinion on this.